zyte_common_items/pages.py

```diff
@@ -133,9 +139,33 @@ class BaseJobPostingPage(
 
 
 class BaseProductPage(BasePage, Returns[Product], HasMetadata[ProductMetadata]):
+    _parsed_price: Optional[Price] = None
+
+    def _get_parsed_price(self):
+        if not hasattr(self, "price"):
+            return None
+        if self._parsed_price is None:
+            self.price
```

Comment:
I think we should also support async price fields, and add tests for them

Reply:
It seems it means that the currencyRaw field in the base class also becomes async 🙀